not_found_handler.h

```diff
@@ -0,0 +1,18 @@
+#ifndef NOT_FOUND_HANDLER_H
+#define NOT_FOUND_HANDLER_H
+
+#include <string>
+
+#include "request_handler.h"
+#include "config_parser.h"
+#include "request.h"
+
+class NotFoundHandler : public RequestHandler {
+public:
+	
+	virtual std::string GenerateResponse(std::unique_ptr<Request> &request) const;
+	
```

Comment:
TODO:
change std::string to Response class,
rename GenerateResponse to HandleRequest
constructor magic